samza-core/src/test/java/org/apache/samza/clustermanager/TestContainerProcessManager.java

```diff
@@ -71,7 +71,7 @@
       put("cluster-manager.container.retry.count", "1");
       put("cluster-manager.container.retry.window.ms", "1999999999");
       put("cluster-manager.allocator.sleep.ms", "1");
-      put("cluster-manager.container.request.timeout.ms", "2");
+      put("cluster-manager.container.request.timeout.ms", "5000");
```

Comment:
Any reason for this change?
If there are "expiry-based" tests, this will cause build time to increase.